PukingPredator/Assets/Scripts/Item.cs

```diff
@@ -41,21 +42,36 @@ public class Item : MonoBehaviour
     private GameObject instance;
 
     /// <summary>
-    /// The entity that is holding the item.
+    /// The mass of an object. This is based on its size.
     /// </summary>
-    private GameObject owner;
+    public float mass => sizeToMass[size];
 
     /// <summary>
-    /// If the item is entering, leaving, or sitting in the inventory.
+    /// The entity that is holding the item.
     /// </summary>
-    public ItemState state { get; private set; } = ItemState.beingConsumed;
+    private GameObject owner;
 
     /// <summary>
     /// Size of the item, affects mass of the player
     /// </summary>
     /// TODO: vary sizes
     public ItemSize size { get; private set; } = ItemSize.small;
 
+    /// <summary>
+    /// The mappings from sizes to masses
+    /// </summary>
+    private Dictionary<ItemSize, float> sizeToMass = new()
```

Comment:
Could we potentially just set the mass variables in each specific prefab? If we used sizeToMass, wouldn't we still have to set the size value, to convert to mass? Why would that be better than setting the mass variable directly?

Reply:
that dictionary is just there for now with the (naive) assumption that everything of one size will have the same mass. that can easily be updated though, the sizeToMass is private anyways, code outside of this class should be asking for the computed mass property which can be changed so that each object can have its own mass. then this dictionary will become useless and can just be removed